plugins/modules/aci_l3out_logical_node_profile.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l3out_logical_node_profile
+short_description: Manage Layer 3 Outside (L3Out) logical node profiles (l3extLNodeP:lnodep)
+description:
+- Manage Layer 3 Outside (L3Out) logical node profiles on Cisco ACI fabrics.
+options:
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, name ]
+  description:
+    description:
+    - Description for the node profile.
+    type: str
+    aliases: [ descr ]
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+  dscp:
+    description:
+    - The target Differentiated Service (DSCP) value.
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, CS0, CS1, CS2, CS3, CS4, CS5, CS6, CS7, EF, VA, unspecified ]
+    aliases: [ target ]
```

Comment:
I would use target_dscp and not just dscp as it is very vague and could one day conflict with a generic parameter.